Source/WebCore/style/StyleAdjuster.cpp

```diff
@@ -968,6 +968,14 @@ void Adjuster::adjustForSiteSpecificQuirks(RenderStyle& style) const
             style.setUserSelect(UserSelect::None);
     }
 
+#if PLATFORM(IOS_FAMILY)
+    if (m_document->quirks().needsGoogleMapsScrollingQuirk()) {
+        static MainThreadNeverDestroyed<const AtomString> className("PUtLdf"_s);
+        if (m_element == m_document->body() && const_cast<Element*>(m_element.get())->classList().contains(className))
```

Comment:
I think you can avoid this const_cast by using Element's `hasClassName` method? (just make sure you grab ElementInlines.h)